src/js/pensions/components/FullNameField.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 
 export default function FullNameField({ formData }) {
-  const { first, middle, last, suffix } = formData;
+  const { first, middle, last, suffix } = formData.fullName || formData;
```

Comment:
Is this component being used by something other than military history/planned to be used elsewhere? Is the `formData.fullName` option ever begin used currently?